packages/gatsby-cli/package.json

```diff
@@ -73,9 +74,10 @@
     "directory": "packages/gatsby-cli"
   },
   "scripts": {
-    "build": "babel src --out-dir lib --ignore \"**/__tests__\"",
+    "build": "babel src --out-dir lib --ignore \"**/__tests__\" && npm run showPostInstall",
     "prepare": "cross-env NODE_ENV=production npm run build",
-    "watch": "babel -w src --out-dir lib --ignore \"**/__tests__\""
+    "watch": "babel -w src --out-dir lib --ignore \"**/__tests__\"",
+    "showPostInstall": "node scripts/postinstall.js"
```

Comment:
I did this because sometimes the `postinstall` script was running after the `build` or other scripts. But I agree this is the standard way to do it.

Thanks for taking a look at it! :smiley_cat: